Readme.md

```diff
@@ -42,17 +45,3 @@ Icons:
 ### Requirements
 
 Requires PopClip 2022.12 and an OpenAI Platform account.
-
-### Links
-
-- [GitHub Source](https://github.com/hirakujira/ChatGPT-Grammar-Check-PopClip-Extension)
```

Comment:
We should put this link in the `Acknowledgements` section.

Reply:
Done